src/components/FormAndSuggestionsContainer.js

```diff
@@ -15,14 +15,25 @@ export default function FormAndSuggestionsContainer() {
     setSearchDestination(event.target.value);
   }
 
+  function clearInputDestinatio(e) {
+    e.preventDefault();
+    setSearchDestination('');
+  }
+  function clearInputFrom(e) {
+    e.preventDefault();
+    setSearchFrom('');
+  }
+
   return (
     <>
-      <div className="w-1/4 boxshadow">
+      <div className="w-1/4 boxshadow absolute z-10 rounded-lg bg-white">
```

Comment:
this absolute positioning is irrelevant here. It should be added into home.js when you link the components together.